### PR TITLE
bugfix: Also pass required to textarea element

### PR DESCRIPTION
**Pull request by Pruxis**
## PR Checklist

This PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [x] A changelog entry has been added to CHANGELOG.md if necessary

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] react application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/).
 ### Fixed
 - `Autocomplete` Fixed an issue where re-entering the autocomplete field would trigger an async search with empty query
 - `Autocomplete` Fixed an issue where debounce was using old query values
+- `Textarea` Fixed an issue where required property was not being passed properly
```
Comment (TriangleJuice) [inline CHANGELOG.md:19]: This should go under `[Unreleased]` ;-)

File: packages/form/src/Textarea/Textarea.spec.js [hunk 2]
```diff
@@ -12,4 +12,10 @@ describe('Textarea', () => {
       const component = shallow(<Textarea qa="id-1234" />);
       expect(component.props()).toHaveProperty('data-qa', 'id-1234');
     });
+  
+
+    test('Textarea should contain a reqiured attribute', () => {
```
Comment (TriangleJuice) [inline packages/form/src/Textarea/Textarea.spec.js:17]:
```suggestion
    test('Textarea should contain a required attribute', () => {
```

**Review by TriangleJuice — APPROVED** [review]
Please address small issues. Code is ok!

**Review by MichaelCastiau — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Pruxis) [reply CHANGELOG.md:19]: Ah oke I thought this version was unreleased

**Review by vademo — APPROVED** [review]

**Pull request merged**